rdrf/rdrf/views/form_view.py

```diff
@@ -975,6 +1062,9 @@ def _build_context(self, **kwargs):
         for s in remove_sections:
             sections.remove(s)
 
+        request.session['has_randomised_values'] = bool(randomised)
+        request.session['randomised_values'] = randomised
```

Comment:
Can we key the `randomised_values` by the form id so we won't have surprises down the track in case the user opens multiple forms?

Also, can we clear out these values from the session after the POST is successful?